### PR TITLE
Graphiq Nginx: adds the optional use of limit_conn_zone

### PR DESCRIPTION
**Pull request by jadametz**
- `limit_conn_zone` is defined within the Nginx config regardless of its optional use
- If a container has a defined value for `CONNECTION_ZONE_LIMIT` the rate limiting will be enabled with the variables value as the number of connections allowed from an IP at once

### CODE REVIEW

**Comment by jadametz** [comment]
For reference: http://nginx.org/en/docs/http/ngx_http_limit_conn_module.html

File: graphiq-nginx.tmpl [hunk 1]
```diff
@@ -47,6 +48,9 @@ server {
             {{/* Only create one location per VIRTUAL_HOST */}}
             {{ if eq $break 0}}
         location {{ coalesce $container.Env.VIRTUAL_LOCATION "/" }} {
+            {{ if $container.Env.CONNECTION_ZONE_LIMIT }}
+            limit_conn conn_per_server {{ $container.Env.CONNECTION_ZONE_LIMIT }};
```
Comment (jportwood) [inline graphiq-nginx.tmpl]: What is `conn_per_server`?
The documentation has `limit_conn perserver 100;` --> `perserver` vs `conn_per_server`

Comment (jadametz) [inline graphiq-nginx.tmpl]: It's just based on the value assigned to `zone` on L41. Naming the zones allows you to setup difference configurations for difference scenarios.

File: graphiq-nginx.tmpl [hunk 2]
```diff
@@ -47,6 +48,9 @@ server {
             {{/* Only create one location per VIRTUAL_HOST */}}
             {{ if eq $break 0}}
         location {{ coalesce $container.Env.VIRTUAL_LOCATION "/" }} {
+            {{ if $container.Env.CONNECTION_ZONE_LIMIT }}
```
Comment (docnick) [inline graphiq-nginx.tmpl]: so if this environment variable is left unset, there will be no rate limiting, correct?

Comment (docnick) [inline graphiq-nginx.tmpl]: add comments to explain what is going on here.

**Pull request merged**